wpeview/src/main/cpp/Runtime/WKSettings.cpp

```diff
@@ -69,6 +69,14 @@ void nativeSetDeveloperExtrasEnabled(JNIEnv* /*env*/, jobject /*obj*/, jlong wkW
     }
 }
 
+void nativeSetDisableWebSecurity(JNIEnv* /*env*/, jobject /*obj*/, jlong wkWebViewPtr, jboolean disable) noexcept
```

Comment:
I am not much of a fan of attaching the method to a web view, because it gives the impression that changing the setting will affect only that one web view, when in reality it's going to affect web views that share the same settings object. If settings are not explicitly set when constructing a web view, that means changing the default settings, which may affect all new web views instantiated afterwards.

I won't make you change this before landing, but at least add a `FIXME`/`TODO` somewhere to improve this later on. Creating an issue in the bug tracker would be a nice bonus, too.

Reply:
I understand your point, but in this case this is the same pattern used for all previous settings. So I prefer using the same pattern than creating an exception just for the "DisableWebSecurity" settings.

Nevertheless, I agree that we should review completely how settings are managed to stick to the WebKit native approach. I'll add a FIXME comment in WKSettings.cpp to think about it in the future.